packages/waku/src/lib/handlers/dev-worker-api.ts

```diff
@@ -1,4 +1,7 @@
-import type { Worker as WorkerOrig } from 'node:worker_threads';
+import type {
+  TransferListItem,
+  Worker as WorkerOrig,
```

Comment:
btw, I recently use `WorkerType` for the naming convention instead of `WorkerOrig`. What do you think?